test/Microsoft.OpenApi.Readers.Tests/V3Tests/OpenApiCallbackTests.cs

```diff
@@ -88,7 +88,8 @@ public void ParseAdvancedCallbackWithReferenceShouldSucceed()
 
                 var callback = subscribeOperation.Callbacks["simpleHook"];
 
-                context.ShouldBeEquivalentTo(new OpenApiDiagnostic());
+                context.ShouldBeEquivalentTo(
```

Comment:
>context [](start = 16, length = 7)

If you have some time, would you mind fixing these context -> diagnostic? If not, it's okay. We can do a big cleanup later.